lib/ansible/modules/network/eos/eos_user.py

```diff
@@ -46,6 +46,11 @@
       - The username to be configured on the remote Arista EOS
         device.  This argument accepts a stringv value and is mutually
         exclusive with the C(users) argument.
+  password:
+    description:
+      - The password to be configured on the remote Arista EOS device. The
+        password needs to be provided in clear and it will be encrypted
+        on the device.
```

Comment:
Would you mind adding another line stating that this `password` is not same as `provider password` to make the Doc more clear to users? just like this https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/eos/eos_user.py#L49